### DISCUSSION TITLE
What does `$meta` mean?

### DISCUSSION THREAD

**Discussion by secsilm** [milvus-io/milvus#34536] [Q&A and General discussion] [answered]
Below is a piece of code generated by attu. What does `$meta` mean? A special symbol representing all dynamic fields?

```python
# python search code
res = client.search(
  collection_name="en_tmp", # Collection name
  data=query_vector, # Replace with your query vector
  search_params={
    "metric_type": "COSINE",
    "params": {}, # Search parameters
  }, # Search parameters
  limit=50, # Max. number of search results to return
  output_fields=["news_id","country","publish_time","$meta"], # Fields to return in the search results
  consistency_level="Strong"
)
```

**Comment by yhmo** [comment, marked as answer]
$meta is the name of the dynamic field.
If you turn on the "enable_dynamic_field", the collection will have a build-in JSON type field whose name is "$meta" to store all the dynamic fields values.
`schema = CollectionSchema(fields=fields, enable_dynamic_field=True)`
